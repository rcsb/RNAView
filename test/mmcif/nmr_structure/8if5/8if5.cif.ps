%!PS-Adobe-2.0
%%Title: A postscript generated from a RNAML file.
%%CreationDate: Mon Dec 11 10:28:38 2023
%%Orientation: Portrait
%%BoundingBox:    196   231   417   561

/Times-Bold findfont 12  scalefont setfont


/Times-Bold findfont 13  scalefont setfont    %%set font for (A U G C T)

%% stacking diagram widths for base-pairs 1 & 2
/W1 {1 setlinewidth} bind def
/W2 {1.5 setlinewidth} bind def
/W3 {2 setlinewidth} bind def
/W4 {3 setlinewidth} bind def

%% minor and major grooves filling color saturation
/MINOR_SAT 0.9 def
/MAJOR_SAT 0.1 def
/OTHER_SIDES {0 0 1 sethsbcolor fill} bind def

%% defineing geometry shapes
/NP {newpath} bind def
/CIRCLE {0 360 arc closepath} bind def                        %% circle 
/TRIANGLE {moveto lineto lineto closepath} bind def           %% triangle
/SQUARE {moveto lineto lineto lineto closepath} bind def      %% square
/LINE {moveto lineto stroke} bind def                         %% line
/DASHLINE { moveto lineto [2 4] 0 setdash stroke } bind def %% dashline
/DOTLINE  {1 0 1 setrgbcolor} bind def                        %% dotline
/Dw {1 setlinewidth} bind def
/FB {setgray fill} bind def
/R6 {moveto lineto lineto lineto lineto lineto closepath} bind def
/R9 {moveto lineto lineto lineto lineto lineto
     lineto lineto lineto closepath} bind def

%% line drawing colors for ACGITU & others
/Al {0.0000 1.00 1.00 sethsbcolor} bind def
/Cl {0.1667 1.00 1.00 sethsbcolor} bind def
/Gl {0.3333 1.00 1.00 sethsbcolor} bind def
/Il {0.3333 1.00 0.57 sethsbcolor} bind def
/Tl {0.6667 1.00 1.00 sethsbcolor} bind def
/Ul {0.5000 1.00 1.00 sethsbcolor} bind def
/Xl {0.0000 0.00 0.00 sethsbcolor} bind def
/XX {0.2000 0.20 0.20 sethsbcolor} bind def
/FILLBLACK {0.0000 0.00 0.00 sethsbcolor fill} bind def

%% minor groove filling colors for ACGITU & others
/Am {0.0000 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Cm {0.1667 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Gm {0.3333 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Im {0.3333 MINOR_SAT 0.57      sethsbcolor fill} bind def
/Tm {0.6667 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Um {0.5000 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Xm {0.0000 0.00      MAJOR_SAT sethsbcolor fill} bind def

%% major groove filling colors for ACGITU & others
/AM {0.0000 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/CM {0.1667 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/GM {0.3333 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/IM {0.3333 MAJOR_SAT 0.57      sethsbcolor fill} bind def
/TM {0.6667 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/UM {0.5000 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/XM {0.0000 0.00      MINOR_SAT sethsbcolor fill} bind def

%% define line width, line join style & cap style (1 means round)
1 setlinewidth 1 setlinejoin 1 setlinecap
   216   251 translate

/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
125.38 286.48  moveto (C) center show 
125.38 257.32  moveto (A) center show 
125.38 228.15  moveto (C) center show 
125.38 198.98  moveto (G) center show 
125.38 169.82  moveto (T) center show 
150.25 154.35  moveto (G) center show 
168.91 131.80  moveto (T) center show 
179.45 104.48  moveto (T) center show 
180.76 75.23  moveto (G) center show 
172.72 47.07  moveto (T) center show 
156.16 22.93  moveto (C) center show 
132.79 5.30  moveto (T) center show 
105.02 -4.00  moveto (C) center show 
75.74 -4.00  moveto (T) center show 
47.98 5.30  moveto (C) center show 
24.60 22.93  moveto (T) center show 
8.04 47.07  moveto (G) center show 
0.00 75.23  moveto (T) center show 
1.32 104.48  moveto (G) center show 
11.85 131.80  moveto (T) center show 
30.52 154.35  moveto (C) center show 
55.38 169.82  moveto (T) center show 
55.38 198.98  moveto (C) center show 
55.38 228.15  moveto (G) center show 
55.38 257.32  moveto (T) center show 
55.38 286.48  moveto (G) center show 
0.005 setlinewidth 1 setlinejoin 1 setlinecap
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 125.38 296.48 moveto (5') center show grestore
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 55.38 296.48 moveto (3') center show grestore
Il 125.38 290.48 125.38 261.32 LINE
 Il 125.38 261.32 125.38 232.15 LINE
 Il 125.38 232.15 125.38 202.98 LINE
 Il 125.38 202.98 125.38 173.82 LINE
 Il 125.38 173.82 150.25 158.35 LINE
 Il 150.25 158.35 168.91 135.80 LINE
 Il 168.91 135.80 179.45 108.48 LINE
 Il 179.45 108.48 180.76 79.23 LINE
 Il 180.76 79.23 172.72 51.07 LINE
 Il 172.72 51.07 156.16 26.93 LINE
 Il 156.16 26.93 132.79 9.30 LINE
 Il 132.79 9.30 105.02 0.00 LINE
 Il 105.02 0.00 75.74 0.00 LINE
 Il 75.74 0.00 47.98 9.30 LINE
 Il 47.98 9.30 24.60 26.93 LINE
 Il 24.60 26.93 8.04 51.07 LINE
 Il 8.04 51.07 0.00 79.23 LINE
 Il 0.00 79.23 1.32 108.48 LINE
 Il 1.32 108.48 11.85 135.80 LINE
 Il 11.85 135.80 30.52 158.35 LINE
 Il 30.52 158.35 55.38 173.82 LINE
 Il 55.38 173.82 55.38 202.98 LINE
 Il 55.38 202.98 55.38 232.15 LINE
 Il 55.38 232.15 55.38 261.32 LINE
 Il 55.38 261.32 55.38 290.48 LINE
 /Times findfont 8 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Tl 44.38 257.32 moveto (25) center show grestore
gsave Tl 134.38 169.82 moveto (5) center show grestore
gsave Tl 183.58 41.97 moveto (10) center show grestore
gsave Tl 42.39 -5.32 moveto (15) center show grestore
gsave Tl 1.50 137.86 moveto (20) center show grestore
NP W1
 Xl 117.38 288.48 63.38 288.48 LINE
 Xl 117.38 292.48 63.38 292.48 LINE
 NP W1
 Xl 117.38 261.32 63.38 261.32 LINE
 NP W1
 Xl 117.38 230.15 63.38 230.15 LINE
 Xl 117.38 234.15 63.38 234.15 LINE
 NP W1
 Xl 117.38 200.98 63.38 200.98 LINE
 Xl 117.38 204.98 63.38 204.98 LINE
 NP W1
 Xl 117.38 173.82 93.38 173.82 LINE
 Xl 90.38 173.82 3.00 CIRCLE 
 gsave FILLBLACK grestore stroke
 Xl 87.38 173.82 63.38 173.82 LINE
 NP W1
 Xl 170.55 127.97 174.22 110.45 LINE
 Xl 177.16 111.06 171.29 109.83 175.45 104.58 TRIANGLE
 gsave FILLBLACK grestore stroke
 Xl 175.45 104.58 179.12 87.06 LINE
 NP W1
 Xl 163.38 130.01 110.52 74.72 LINE
 Xl 108.45 72.55 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 106.37 70.38 53.51 15.08 LINE
 NP W1
 Xl 174.62 73.73 55.99 11.26 LINE
 Xl 172.75 77.27 54.13 14.80 LINE
 NP W1
 Xl 168.20 44.48 168.97 45.60 LINE
 Xl 168.61 39.78 163.67 43.17 168.97 45.60 TRIANGLE
 gsave FILLBLACK grestore stroke
 Xl 166.14 41.48 162.75 36.53 LINE
 Xl 165.22 34.83 162.39 30.71 157.44 34.10 160.27 38.23 SQUARE
 gsave FILLBLACK grestore stroke
 Xl 159.92 32.41 160.69 33.53 LINE
 NP W1
 Xl 150.01 32.43 9.33 106.52 LINE
 Xl 148.15 28.89 7.46 102.98 LINE
 NP W1
 Xl 125.20 6.76 126.49 7.19 LINE
 Xl 122.70 2.76 120.80 8.45 126.49 7.19 TRIANGLE
 gsave FILLBLACK grestore stroke
 Xl 121.75 5.60 116.06 3.70 LINE
 Xl 117.01 0.85 112.27 -0.74 110.37 4.95 115.11 6.54 SQUARE
 gsave FILLBLACK grestore stroke
 Xl 111.32 2.11 112.61 2.54 LINE
 NP W1
 Xl 98.88 5.50 16.05 49.12 LINE
 Xl 97.01 1.96 14.19 45.58 LINE
 Al W2 145.46 154.74 12.83 54.69 gsave  DASHLINE grestore
 Al W2 152.01 31.27 34.67 154.02 gsave  DASHLINE grestore
 Al W2 177.29 114.08 171.07 130.20 gsave  DASHLINE grestore
 Al W2 180.50 85.22 179.72 102.48 gsave  DASHLINE grestore
 
 showpage 
