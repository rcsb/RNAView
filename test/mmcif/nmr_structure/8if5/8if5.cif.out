PDB data file name: mmcif-test/nmr_structure/8if5/8if5.cif
-----------------------------------------------------------
CRITERIA USED TO GENERATE BASE-PAIR: 
  3.40 --> upper H-bond length limits (ON..ON).
 26.00 --> max. distance between paired base origins.
  2.50 --> max. vertical distance between paired base origins.
 65.00 --> max. angle between paired bases [0-90].
  5.40 --> min. distance between RN9/YN1 atoms.
  8.00 --> max. distance criterion for helix break[0-12]
-----------------------------------------------------------
BASE-PAIR INSTRUCTIONS: 
Column 1 is rnaview assigned base numbers n1_n2, start from 1.
Column 2 & 3 are chain ID & residue number in input PDB file.
Column 4 is for base pair. The left & right are the bases as 
         identified by column 2 & 3 and 5 & 6.
Column 5 & 6 are residue number & chain ID in input PDB file.
Column 7 is for base pair annotation. The standard Watson-Crick
         (W.C.) pairs are annotated as -/- (AU,AT) or +/+ (GC).
         Other pairs are annotated as Leontis_Westhof Classification.
         The three edges: W(Watson-Crick); H(Hoogsteen); S(suger).
         e.g. W/H means the pair is edge of Watson-Crick & Hoogsteen.
Column 8 is glycosidic bond orientation (either cis or trans).
         e.g. 'W/H cis' means the pair is interaction on Watson-Crick
         and Hoogsteen side, glycosidic bond orientation is 'cis'.
Column 9 corresponds to Saenger Classification.

Other columns: 
        Syn sugar-base conformations are annotated as (syn).
        Stacked base pairs are annotated as (stack).
        Non-identified edges are annotated as (.) or (?)
        Tertiary interactions are marked by (!) in the line.
Reference:
Yang et al (2003) Nucleic Acids Research, Vol31,No13,p3450-3461.
-----------------------------------------------------------
BEGIN_base-pair
     1_26, A:     1 C-G    26 A: +/+ cis         XIX
     2_25, A:     2 A-T    25 A: -/- cis         XX
     3_24, A:     3 C-G    24 A: +/+ cis         XIX
     4_23, A:     4 G-C    23 A: +/+ cis         XIX
     5_22, A:     5 T-T    22 A: W/W cis         XVI
      7_9, A:     7 T-G     9 A: S/S cis         n/a
     7_15, A:     7 T-C    15 A: W/W tran        XVII
     9_15, A:     9 G-C    15 A: +/+ cis         XIX
    10_11, A:    10 T-C    11 A: S/H cis         n/a
    11_19, A:    11 C-G    19 A: +/+ cis         XIX
    12_13, A:    12 T-C    13 A: S/H cis         n/a
    13_17, A:    13 C-G    17 A: +/+ cis         XIX
    18_19, A:    18 T-G    19 A:      stacked
    19_20, A:    19 G-T    20 A:      stacked
     6_17, A:     6 G-G    17 A: W/H tran        !1H(b_b)
    11_21, A:    11 C-C    21 A: W/S cis         !1H(b_b)
      7_8, A:     7 T-T     8 A: S/. cis         !(s_s)
      8_9, A:     8 T-G     9 A: S/S cis         !(s_s)
END_base-pair

Summary of triplets and higher multiplets
BEGIN_multiplets
7_9_15_| [1 3]  A: 7 T  +  A: 9 G  +  A: 15 C
10_11_19_| [2 3]  A: 10 T  +  A: 11 C  +  A: 19 G
12_13_17_| [3 3]  A: 12 T  +  A: 13 C  +  A: 17 G
END_multiplets

  The total base pairs =  12 (from   26 bases)
------------------------------------------------
 Standard  WW--cis  WW-tran  HH--cis  HH-tran  SS--cis  SS-tran
        7        1        1        0        0        1        0
  WH--cis  WH-tran  WS--cis  WS-tran  HS--cis  HS-tran
        0        0        0        0        2        0
------------------------------------------------
